obspy/geodetics/base.py

```diff
@@ -315,37 +315,42 @@ def locations2degrees(lat1, long1, lat2, long2):
     Earth. For more accurate values use the geodesic distance calculations of
     geopy (https://github.com/geopy/geopy).
 
-    :type lat1: float
-    :param lat1: Latitude of point 1 in degrees
-    :type long1: float
-    :param long1: Longitude of point 1 in degrees
-    :type lat2: float
-    :param lat2: Latitude of point 2 in degrees
-    :type long2: float
-    :param long2: Longitude of point 2 in degrees
-    :rtype: float
-    :return: Distance in degrees as a floating point number.
+    :type lat1: ``float or :class:`numpy.ndarray```
```

Comment:
Ah sorry - my bad - I intended to use single backticks so github would render it nicely but I screwed up and used two backticks...sorry about that.